src/System.DirectoryServices/src/System/DirectoryServices/PrivilegedConfigurationManager.cs

```diff
@@ -8,21 +8,20 @@ namespace System.Configuration
     using System.Security;
     using System.Security.Permissions;
 
-    [ConfigurationPermission(SecurityAction.Assert, Unrestricted = true)]
     internal static class PrivilegedConfigurationManager
     {
-        internal static ConnectionStringSettingsCollection ConnectionStrings
-        {
-            [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode")]
-            get
-            {
-                return ConfigurationManager.ConnectionStrings;
-            }
-        }
+        //internal static ConnectionStringSettingsCollection ConnectionStrings
```

Comment:
This code was added at some point to work around an OS issue.  Before I just remove it I want to make sure it's not needed anymore.